ts/components/conversation/message/message-content/MessageLinkPreview.tsx

```diff
@@ -59,31 +62,28 @@ export const MessageLinkPreview = (props: Props) => {
       className={classNames('module-message__link-preview')}
       onClick={openLinkFromPreview}
     >
-      {first.image && previewHasImage && isFullSizeImage ? (
-        <ImageGrid attachments={[first.image]} onError={props.handleImageError} />
-      ) : null}
       <div className={classNames('module-message__link-preview__content')}>
-        {first.image && previewHasImage && !isFullSizeImage ? (
+        {previewHasImage ? (
           <div className="module-message__link-preview__image_container">
             <Image
               softCorners={true}
               alt={window.i18n('previewThumbnail', [first.domain])}
-              height={72}
-              width={72}
+              height={100}
```

Comment:
Are we using these image dimensions any where else in the app? Maybe we should make them constants?